main/iptables/iptables-1.8.2-musl-fixes.patch

```diff
@@ -34,12 +34,27 @@
 
  enum {
  	O_STRIP_OPTION = 0,
---- iptables-1.4.12.1.o/include/linux/netfilter_ipv4/ip_tables.h
-+++ iptables-1.4.12.1/include/linux/netfilter_ipv4/ip_tables.h
+--- a/include/linux/netfilter_ipv4/ip_tables.h
++++ b/include/linux/netfilter_ipv4/ip_tables.h
 @@ -16,6 +16,7 @@
  #define _IPTABLES_H
 
  #include <linux/types.h>
 +#include <sys/types.h>
 
  #include <linux/netfilter_ipv4.h>
+
+--- a/extensions/libebt_vlan.c
++++ b/extensions/libebt_vlan.c
+@@ -13,7 +13,11 @@
+ #include <ctype.h>
+ #include <xtables.h>
+ #include <linux/netfilter_bridge/ebt_vlan.h>
++#if defined (__GLIBC__)
+ #include <linux/if_ether.h>
```

Comment:
`linux/if_ether.h` should be a header from `linux-headers` and not related to GLIBC at all. Which makes me wonder why this patch is necessary?

Reply:
Upstream patch is here: https://git.netfilter.org/iptables/commit/?id=51d374ba41ae4f1bb851228c06b030b83dd2092f
Last time there was also a conflict with ebtables: https://github.com/alpinelinux/aports/commit/ad7465682e465adc4d142ffabb09d51b0e159f53